File: An2/MPP/mpp-proiect-java-DeepSmeag/modelModule/src/main/java/contests/model/PersoanaOficiu.java
package contests.model;

import contests.model.entity.Entitate;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@NamedQuery(name = "PersoanaOficiu.findPersoanaOficiu", query = "SELECT p FROM PersoanaOficiu p WHERE p.username = :username")
@Table(name = "persoanaoficiu")
public class PersoanaOficiu extends Entitate<Integer> implements Serializable {
    //    o persoanaOficiu este asociata unui singur oficiu
//    pot fi mai multe persoaneOficiu asociate unui oficiu
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private int id;
    @Basic
    @Column(name = "username")
    private String username;
    @Basic
    @Column(name = "password")
    private String password;
    public PersoanaOficiu() {
        super(null);
    }
    public PersoanaOficiu(Integer id, String username, String password) {
        super(id);
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }
}

File: An2/MPP/mpp-proiect-csharp-DeepSmeag/models/PersoanaOficiu.cs
using mpp_proiect_csharp_DeepSmeag.models.entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace mpp_proiect_csharp_DeepSmeag.models
{

    [Serializable]
    public class PersoanaOficiu : Entitate<int>
    {
        public string username { get; }        

        public string password { get; }

        public PersoanaOficiu(int id, string username, string password) : base(id)
        {
            this.username = username;
            this.password = password;
        }
    }
}
